flink-formats/flink-protobuf/src/test/java/org/apache/flink/formats/protobuf/BigPbProtoToRowTest.java

```diff
@@ -82,9 +82,52 @@ public void testSimple() throws Exception {
                         .putMapField33("key2", "value2")
                         .build();
 
+        String[][] projectedField =
```

Comment:
Can you add a UT to verify your logic without modifying the original UT，This comment applies to all UT，WDYT?

Reply:
> Can you add a UT to verify your logic without modifying the original UT，the comment applies to all UT，WDYT?

@ljw-hit Hi, this is because I changed the constructor of PbRowDataDeserializationSchema, the projectedFields is a mandatory parameter. And for most cases, I created a all field convertor. So it takes the same PB input. 

`public ProtoToRowConverter(
            RowType rowType, PbFormatConfig formatConfig, String[][] projectedFields)
            throws PbCodegenException {`